src/gd_kotlin.cpp

```diff
@@ -35,8 +40,25 @@ jni::JObject to_java_url(jni::Env& env, const String& bootstrapJar) {
     return url;
 }
 
-jni::JObject create_class_loader(jni::Env& env, const String& bootstrapJar) {
-    jni::JObject url = to_java_url(env, bootstrapJar);
+jni::JObject create_class_loader(jni::Env& env, const String& full_jar_path, const jni::JObject& p_parent_loader) {
```

Comment:
parent_class_loader is not being used in the non android path. As I've mentioned before URLClassLoader's constructor accepts a class loader as the second parameter :)